src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -3,20 +3,23 @@ title: 'Quotas & Events'
 sidebar_order: 0
 ---
 
-Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events to be tracked. This number of events is your quota. When an event is accepted by Sentry, it counts toward your quota.
+Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events - errors, attachments, and transactions - to be tracked. Each of these type of events has a quota. When an event is accepted by Sentry, it counts toward your quota for that type of event.
 
 Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.
 
 ## Key Terms
 
 Let’s clarify a few terms to start:
+
 -   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a set of characteristics, called its fingerprint.
--   Issue - an issue is a grouping of similar events, who all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
--   Quota - your quota is the monthly number of events you pay Sentry to track.
+-   Issue - an issue is a grouping of similar events, which all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
+-   Attachment - Attachments are files uploaded in the same request, such as log files. Unless the option to store crash reports is enabled, Sentry will use the files only to create the event, then will drop the files. 
+-   Transaction - A transaction represents a single instance of a service being called to support an operation your want to measure or track, like a page load.
```

Comment:
```suggestion
-   Transaction - A transaction represents a single instance of a service being called to support an operation you want to measure or track, like a page load.
```

Reply:
Thanks for catching the typo. I'm going to keep the ordering as is since folks need to read the preceding definitions to understand the final definition (quota).